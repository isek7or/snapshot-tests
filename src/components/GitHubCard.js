import Card from 'react-bootstrap/Card';
import github from './github.jpg'

function GitHubCard() {
    return (
        <Card style={{ width: '18rem' }}>
            <Card.Img variant="top" src={github} />
            <Card.Body>
                <Card.Title>isek7or</Card.Title>
                <Card.Text>
                    We are young developers with a bright future.
                </Card.Text>
            </Card.Body>
        </Card>
    );
}

export default GitHubCard;