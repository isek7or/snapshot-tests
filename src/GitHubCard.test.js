import { render, screen } from '@testing-library/react';
import GitHubCard from '../src/components/GitHubCard';
import renderer from 'react-test-renderer'

test('snapshot of my github account', () => {
    const tree = renderer.create(<GitHubCard />).toJSON()
    console.log(tree)
    expect(tree).toMatchSnapshot()
  })