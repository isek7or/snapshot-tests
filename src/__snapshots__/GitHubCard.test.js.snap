// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot of my github account 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="github.jpg"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      isek7or
    </div>
    <p
      className="card-text"
    >
      We are young developers with a bright future.
    </p>
  </div>
</div>
`;
